crates/sui-deepbook-indexer/src/migrations/00000000000000_diesel_initial_setup/up.sql

```diff
@@ -178,4 +180,22 @@ CREATE TABLE IF NOT EXISTS sui_error_transactions
     failure_status              TEXT         NOT NULL,
     package                     TEXT         NOT NULL,
     cmd_idx                     BIGINT
-);
\ No newline at end of file
+);
+
+CREATE TABLE IF NOT EXISTS pools
+(
+    id                          SERIAL       PRIMARY KEY,
+    pool_id                     TEXT         NOT NULL,
```

Comment:
is pool_id u256? can we use BIGINT for this, same for other id fields

Reply:
I believe Bridger tried this but said String is what worked. Been copying that since